kura/test/org.eclipse.kura.core.configuration.test/src/main/resources/kura_custom.properties

```diff
@@ -0,0 +1 @@
+kura.primary.network.interface=enp0s3
```

Comment:
Well, I do... As we already discussed, many different names are now possible. If you want it changed, to what? If to eth0, I might as well remove the file.